core/include/scipp/core/transform.h

```diff
@@ -51,6 +52,11 @@ template <class T> struct ValuesAndVariances {
   ValuesAndVariances(T &val, T &var) : values(val), variances(var) {
     expect::sizeMatches(values, variances);
   }
+  ValuesAndVariances &operator=(std::pair<T, T> &&data) {
```

Comment:
Where exactly does move assignment help?

Reply:
It is required to make the assignment in https://github.com/scipp/scipp/blob/2e156538bddbadb09016834430bd1eb8292ebbd0/core/include/scipp/core/transform.h#L227-L230 work. The key is not the assignment from move, but the transition from `pair` to `ValuesAndVariances`.